_config.yml

```diff
@@ -81,26 +81,24 @@ footer:
       - label: Mailing List
         url: "https://groups.google.com/forum/#!forum/google-containers" 
 
-include: [v1.0, vHEAD]
+include: [v1.1, v1.0]
 
 # Collections (for versioning docs)
 collections:
--v1.0:
-      output:true
-      version:v1.0
--vHEAD:
-      output:true
-      version:vHEAD
+   v1.1:
```

Comment:
No dashes. This is pushed to my fork and works. Ive also got both v1.0 and v1.1 working in my local build (testing the v1.1 setup by using a copy of the v1.0 .html files).

I think you have the option to list individual collection names (and use dashes) but if you add meta data (ie how ive forced/ensure our collections get built and use "output: true"), then you have to remove dashes.

The old dashes were left in there from my early testing/trying to get this working for v1.0. Dashes were used initially because a few sources I used online, including the [Jekyll help docs](http://jekyllrb.com/docs/collections/#step-1-tell-jekyll-to-read-in-your-collection) are a little confusing and have examples both with and without dashes.